components/docs-chef-io/content/automate/servicenow_integration.md

```diff
@@ -0,0 +1,257 @@
+   Note that the Data Feed URL is the ServiceNow application, `FQDN/api/x_chef_automate/asset`. For example,  <https://venxxx.service-now.com/api/x_chef_automate/asset>.
+1. Click the **Test Data Feed** button. This will check that the values are correct and that there is connectivity between Chef Automate and the ServiceNow application. A successful test shows the **service-now automate connectivity passed** message. If there is an error, a message is displayed that will help resolve any connectivity or credentials issues. 
+1. Click **Create Data Feed**. The configuration is saved.
+   
+{{< figure src="/images/automate/snow_integration_create_data_feed.png" alt="Create Data Feed">}}
+## Navigation
+In ServiceNow, the navigation of the application is through the **Chef Automate** menu.
+{{< figure src="/images/automate/snow_integration_navigation.png" alt="Navigation">}}
+
+The **Automate Instances** module allows the user to configure the integration with Chef Automate. The main module for navigation is the **Servers** module. The **Server** module displays a list of servers in the _CMDB_ module. The Chef Automate integration augments the existing CMDB servers and inserts new servers into CMDB. The application uses the ServiceNow discovery IRE (Identification and Reconciliation Engine) when inserting or updating servers.<br  />
+In addition, the application updates the CMDB file systems and software installed tables, and adds related information on the servers form with related data from Chef Automate:
+- Client Runs
+- Attributes
+- Compliance Reports
+### Client Runs
+From a server record, the user can drill down into **Client Run** detail by clicking the name on an individual Client Run. Client run record displays related information for:
+- Client run cookbooks - cookbooks executed during the client run.
+- Client run lists - run lists executed during the client run.
+- Client run recipes - recipes executed during the client run.<br  />
+Full Chef client run details are available for each server. Client runs are also available from the **Client Runs** module.
+### Attributes
+From a server record, the user can drill down into the current server attributes detail by clicking on the attributes record. Full OHAI attributes are available for each server.
+### Compliance Reports
+From a server record, the user can drill down into **Compliance** report detail by clicking the name on an individual **Compliance** report. Compliance report record displays related information for:
+- Compliance report profiles - all profiles executed during the compliance scan.
+- Compliance report results - all results from the compliance scan.
+In addition, the user can drill down into each **Compliance** report profile to view the individual results for each profile.<br  /> Full Chef compliance report details are available for each server. Compliance reports are also available from the **Compliance** report module.
```

Comment:
If we want the line 252 after br to not be part of the list, can give a line between them ?